### PR TITLE
Feature/add label to facets

### PR DESCRIPTION
**Pull request by sonic182**
@carlosescri this works (I copied the bundled files to webcomponents and tested)

I added the "brand" label to the brand facet in the test search engine, the others are empty that's why others are with `''`
image

### CODE REVIEW

**Comment by carlosescri** [comment]
Where all those commits come from? Could you apply from a clean master?

File: src/response.ts [hunk 1]
```diff
@@ -351,6 +356,7 @@ function processTermsFacet(facet: RawTermsFacet): TermsFacet {
 
   return {
     type: 'terms',
+    label: facet.label,
```
Comment (hailem) [inline src/response.ts:359]: Add the visible property too.

**Comment by sonic182** [comment]
> Where all those commits come from? Could you apply from a clean master?

I can't, I tried

File: src/response.ts (hunk 1 above)
Comment (sonic182) [reply src/response.ts:359]: Don't understand

**Comment by sonic182** [comment]
Added visible property @carlosescri @hailem

**Pull request merged**